### DISCUSSION TITLE
Shiny list from serebii

### DISCUSSION THREAD

**Discussion by Rplus** [Rplus/Pokemon-shiny#414] [Q&A]
Does this shiny list from serebii contain all shiny pokemon?
https://www.serebii.net/pokemongo/shiny.shtml
![image](https://github.com/user-attachments/assets/415949f5-c047-4dc4-b17a-cf1665d332bd)

**Comment by BartmanEH** [comment]
it does seem to have them all but I'm not sure how quickly they update this list for releases

**Comment by Rplus** [comment]
There is another shiny list with wiki system
https://bulbapedia.bulbagarden.net/wiki/List_of_Shiny_Pokémon_in_Pokémon_GO

**Reply by BartmanEH** [reply]
this one is a little strange, for example, babies are listed with their evolutions and not as separate dex entry even though they have their own dex entry number:
<img width="868" alt="Screenshot 2024-10-01 at 9 14 55 AM" src="https://github.com/user-attachments/assets/272e3631-2391-405e-8409-69854c1af6c3">
Magby should be on its own with dex entry #240 which is missing in the list.

**Comment by BartmanEH** [comment]
Zamazenta will be released in 3 days. Let's see which of these lists gets updated promptly to include it.

I assume you're looking for a more reliable source for images?

**Comment by BartmanEH** [comment]
> Does this shiny list from serebii contain all shiny pokemon? https://www.serebii.net/pokemongo/shiny.shtml

This one is missing Megas
